pyproject.toml

```diff
@@ -47,10 +47,14 @@ docs = [
 pytest = "^8.3"
 pytest-cov = "^6.0"
 
+
 [tool.semantic_release]
 branch = "main"
-version_toml = "pyproject.toml:project.version"
-version_variable = "src/bluetooth_sensor_state_data/__init__.py:__version__"
+version_toml = ["pyproject.toml:tool.poetry.version"]
```

Comment:
This broke the release workflow after the move to poetry `v2`.
Fixed in https://github.com/Bluetooth-Devices/bluetooth-sensor-state-data/pull/52 with version `1.7.5`.

https://pypi.org/project/bluetooth-sensor-state-data/1.7.5/
https://github.com/Bluetooth-Devices/bluetooth-sensor-state-data/actions/runs/13158368561/job/36720692171